etc/Dockerfile.faucet

```diff
@@ -2,11 +2,13 @@
 
 FROM faucet/python3:5.0.1
 
-RUN apk add -q tcpdump iptables sudo
+RUN apk add -q tcpdump iptables sudo linux-headers
```

Comment:
so, as per other comments, this should be a copy of Dockerfile.faucet into Dockerfile.controller then looks more or less exactly like this file.

Reply:
Same as before. Should I just rename the whole thing to controller so that it makes more sense semantically.

Reply:
Separated.